simulation/test_torrent_status.cpp

```diff
@@ -76,35 +99,82 @@ TORRENT_TEST(status_timers)
 			// to stay within that range
 			if (ticks > 20 * 60 * 60)
 			{
-				// unfortunantly we can't test this many ticks
-				TEST_ERROR("This line should be reached but doesn't");
 				return true;
 			}
 
-			if (ticks == 0){
-				// torrent get ready for seeding on first tick, which means +1s
-				startseed_time = lt::clock_type::now();
+			if(tick_is_in_active_range){
+				// 1 second per tick
+				active_time++;
 			}
 
+			// resume on a tick which is far away from next tick check intervall
+			// because with very low upload limit it takes some time to unchock
+			switch(ticks)
```

Comment:
this code is not very intuitive.. I take it you set the ``tick_interval`` to large values for parts of the test to speed up the simulation, is that right?
another way to speed up the test (at least a little bit) is to pause the torrent